CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Unreleased
 
+* [Issue #321](https://github.com/manheim/terraform-pipeline/issues/172) Allow 'apply' on branches other than master for some environments
```

Comment:
Can the Issue number and link be corrected?  Both should point to Issue #25 - https://github.com/manheim/terraform-pipeline/issues/25